perception/image_projection_based_fusion/launch/roi_cluster_fusion.launch.xml

```diff
@@ -18,6 +18,7 @@
   <arg name="input/camera_info7" default="/camera_info7"/>
   <arg name="input/clusters" default="clusters"/>
   <arg name="output/clusters" default="labeled_clusters"/>
+  <arg name="param_path" default="$(find-pkg-share image_projection_based_fusion)/config/roi_fusion.param.yaml"/>
```

Comment:
```suggestion
  <arg name="sync_param_path" default="$(find-pkg-share image_projection_based_fusion)/config/roi_sync.param.yaml"/>
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2175/commits/7807a448f3c772d49ebe50bc6f5fe3863ead6a64